db/migrate/20160127035213_create_seasons.rb

```diff
@@ -3,5 +3,7 @@ def change
     create_table :seasons do |t|
       t.datetime :start_date, null: false
     end
+
+    add_index :seasons, :start_date, unique: true
```

Comment:
I'd advise against editing old migrations. This will cause a lot of pain for anyone else working on the project. Even though in practice you're the only one, it's a bad habit to get into.